crates/swc_plugin_backend_wasmer/benches/ecma_invoke.rs

```diff
@@ -44,26 +44,25 @@ fn plugin_group(c: &mut Criterion) {
 }
 
 fn bench_transform(b: &mut Bencher, plugin_dir: &Path) {
+    let runtime = Arc::new(WasmerRuntime);
```

Comment:
I think we may need to use a separate crate in the future PRs to avoid duplicating all the testing logics and benchmarking logics